pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[build-system]
+requires = [
+    "hatchling>=1.13.0",
+    "hatch-vcs>=0.3.0",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "pyshaper"
+dynamic = ["version"]
+description = "framework for defining, building, and evaluating generalized shape observables for collider physics"
+readme = "README.md"
+license = { text = "MIT" }  # SPDX short identifier
+requires-python = ">=3.7"
+authors = [
+    { name = "Rikab Gambhir", email = "rikab@mit.edu" },
+]
+keywords = [
+    "shape observables",
+    "jet physics",
+]
+classifiers = [
```

Comment:
These metadata are PyPI "trove classifiers" which are searchable topics/keywords that people can browse on PyPI.